src/main/java/com/training/istasenka/annotation/DoIgnore.java

```diff
@@ -0,0 +1,14 @@
+package com.training.istasenka.annotation;
+
+import org.mapstruct.Qualifier;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Qualifier
+@Target(ElementType.METHOD)
+@Retention(RetentionPolicy.CLASS)
+public @interface DoIgnore {
```

Comment:
What is the purpose of that annotation? Cannot find any handler

Reply:
the annotation is used for mapstruct converters with ambiguous methods. Some thing like that https://mapstruct.org/faq/#ambiguous